src/main/java/com/cloudbees/jenkins/plugins/BitBucketTrigger.java

```diff
@@ -120,27 +107,27 @@ public void run() {
      * Returns the file that records the last/current polling activity.
      */
     public File getLogFile() {
-        return new File(job.getRootDir(),"bitbucket-polling.log");
+        return new File(job.getRootDir(), "bitbucket-polling.log");
```

Comment:
Unrelated change to the PR - even if it is wrong I prefer not to modify this and correct it on a separate PR